features/checkout/being_able_to_modify_remaining_steps.feature

```diff
@@ -0,0 +1,30 @@
+Feature: Remember where users actually have left checkout
+    In order to get back to the right checkout step after leaving
+    As a Customer
+    I want to be able to finish checkout process
+
+    Background:
+        Given the store operates on a single channel in "United States"
+        And the store has a product "PHP T-Shirt" priced at "$19.99"
+        And the store ships everywhere for free
+        And the store allows paying offline
+        And I am a logged in customer
+
+    @todo
+    Scenario: Placing an order after navigating to to the past steps of a checkout
```

Comment:
Scenarios should be more specific: `Placing an order after moving back from the checkout summary to the addressing step but without any address modification`